modules/nupac.nu

```diff
@@ -329,19 +352,19 @@ export def "nupac upgrade" [
         if ($to-upgrade|empty?) {
             print "No upgrades found"
         } else {
-            print ($to-upgrade | select name version)
```

Comment:
same goes for L343 `ignore-self` that is used only in L352 `if`

*sorry for the long block but i couldn't comment just that line*

Reply:
```nu
    let ignore-self = (get-flag-value $ignore-self "NUPAC_IGNORE_SELF")
    if (($packages|length) > 0 or $all) {
        let to-upgrade = (
            (get-packages $packages $all)
            |where name not-in (get-ignored)
            |where name != (if $ignore-self {"nupac"} else {""})
        )
```

Reply:
anyway, gonna remove it and we can revert commit if needed